config/betterquesting/DefaultQuests/Quests/AndSoItBegins-AAAAAAAAAAAAAAAAAAAAAA==/MoreUsefulIC2Cro-AAAAAAAAAAAAAAAAAAAGRw==.json

```diff
@@ -16,41 +16,36 @@
       "isMain:1": 0,
       "simultaneous:1": 0,
       "icon:10": {
-        "id:8": "gregtech:gt.metatool.01",
+        "id:8": "IC2:itemCropSeed",
```

Comment:
this looks like something went very wrong here. not sure what though

Reply:
I changed the icon and quest since the sense doesn't exist anymore. So that's why it's a seed instead of a sense.

Reply:
oh wait or is this not meant to be a conversion but a full redo of the quest? XD

Reply:
The flint tools are a conversion, the sense quest was a redo since it didn't make sense (:P) anymore but was tied into requirements of other quests so just wrote some waffle and changed it to a checkbox. Also reduced the reward because of that. It's a minor change but makes it work now.